src/routes/cook.js

```diff
@@ -1,14 +1,119 @@
+/**
+ * @swagger
+ * tags:
+ *   name: Cooks
+ *   description: Cook management API for adding, updating, deleting and getting cooks.
+ */
 const express = require('express');
 
 const router = express.Router();
 
 const cookController = require('../controllers/cookController');
 const { validCook } = require('../validation/validator');
 
+/**
+ * @swagger
+ * /api/cook:
+ *   get:
+ *     summary: Get all cooks
+ *     tags: [Cooks]
+ *     responses:
+ *       200:
+ *         description: Success
+ */
 router.get('/', cookController.getAllCooks);
-router.post('/add',cookController.addCook);
+
+/**
+ * @swagger
+ * /api/cook/add:
+ *   post:
+ *     summary: Add a new cook
+ *     tags: [Cooks]
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/Cook'
```

Comment:
you should add cook in the schema of the swagger.js file. Look at the file there is schema and in there there is dish_add and dish_update, you should have those for cook add and cook update. please solve this.